gpkit/tests/t_keydict.py

```diff
@@ -39,12 +39,6 @@ def test_failed_getattr(self):
         self.assertEqual(res, 5)
         self.assertIn("waldo", kd)
 
-    def test_dictlike(self):
-        kd = KeyDict()
-        kd["a string key"] = "a string value"
-        self.assertTrue(isinstance(kd, dict))
-        self.assertEqual(list(kd.keys()), ["a string key"])
-
```

Comment:
we don't really have a need to target using KeyDicts for things without a `keys` attribute